### PR TITLE
Add dpordomingo as a maintainer of the web app

### PR DESCRIPTION
**Pull request by dpordomingo**
Adding myself as the web app maintainer

### CODE REVIEW

**Comment by dpordomingo** [comment]
@se7entyse7en is this ok? should I add anyone else? maybe you?

**Comment by se7entyse7en** [comment]
> @se7entyse7en is this ok? should I add anyone else? maybe you?

No worries for now 👍

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1 @@
+David Pordomingo <david@sourced.tech> (@dpordomingo)
```
Comment (se7entyse7en) [inline MAINTAINERS]: athenian 😅

Comment (dpordomingo) [reply MAINTAINERS]: it never happened.

**Pull request merged**